src/Core/Settings.h

```diff
@@ -351,6 +351,8 @@ class IColumn;
     M(Bool, opentelemetry_trace_processors, false, "Collect OpenTelemetry spans for processors.", 0) \
     M(Bool, prefer_column_name_to_alias, false, "Prefer using column names instead of aliases if possible.", 0) \
     M(Bool, allow_experimental_analyzer, false, "Allow experimental analyzer", 0) \
+    M(Bool, allow_experimental_get_http_header, false, " experimental function getHTTPHeader", 0) \
+    M(String, headers_forbidden_to_return_list, "", "Comma separated list of http header names that will not be returned by function getHTTPHeader.", 0) \
```

Comment:
these can't be user settings but server settings so user can't modify them per query

Reply:
`get_client_http_header_forbidden_headers`